manticore/core/state.py

```diff
@@ -364,3 +364,6 @@ def mem(self):
 
     def _init_context(self):
         self.context['branches'] = dict()
+
+    def generate_testcase(self, name, message='State generated testcase'):
+        self.publish('state_generate_inputs', message=message, name=name)
```

Comment:
I think the `state_generate_inputs` event should order its args as `(name, message)`, in the same order as the `generate_testcase` api.